src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -509,34 +509,18 @@ export default class ResultsViewOncoprint extends React.Component<
             get annotateDriversHotspotsDisabled() {
                 return !getServerConfig().show_hotspot;
             },
+            get annotateDriversCosmicDisabled() {
+                return !getServerConfig().show_hotspot;
+            },
```

Comment:
Is this intentional? Feels like we shouldn't be referring to `show_hotspot` here.

Reply:
Good catch @onursumer.  I had put this in to test something but forgot to get rid.  It wasn't referenced anywhere.